packages/icons/src/loading-icons.md

```diff
@@ -119,24 +119,24 @@ importing icon modules for you**.  Let's take a look at this required configurat
         return (
             size === IconSize.STANDARD
                 ? await import(
-                    /* webpackChunkName: "blueprint-icons-16px" */
-                    /* webpackInclude: /\.js$/ */
-                    /* webpackMode: "lazy" */
-                    `@blueprintjs/icons/lib/esm/generated/16px/paths/${name}`
-                )
+                      /* webpackChunkName: "blueprint-icons-16px" */
+                      /* webpackInclude: /\.js$/ */
+                      /* webpackMode: "lazy" */
+                      `@blueprintjs/icons/lib/esm/generated/16px/paths/${name}`
+                  )
                 : await import(
-                    /* webpackChunkName: "blueprint-icons-20px" */
-                    /* webpackInclude: /\.js$/ */
-                    /* webpackMode: "lazy" */
-                    `@blueprintjs/icons/lib/esm/generated/20px/paths/${name}`
-                )
+                      /* webpackChunkName: "blueprint-icons-20px" */
+                      /* webpackInclude: /\.js$/ */
+                      /* webpackMode: "lazy" */
+                      `@blueprintjs/icons/lib/esm/generated/20px/paths/${name}`
+                  )
         ).default;
     };
 
     Icons.setLoaderOptions({ loader: lazyLoader });
     ```
 
-4. Load some icon paths up front (dynamically) with network requests, and the rest lazily/on-demand.
+5. Load some icon paths up front (dynamically) with network requests, and the rest lazily/on-demand.
```

Comment:
Why the number change here?

Reply:
Hrm, I think Prettier did this. Will look into it

Reply:
Oh, it's because we have two "4."s in the markdown (but the docs correct for this when published)

https://github.com/palantir/blueprint/blob/c8570835d81382313c2bcf3a09aafe614f3cd854/packages/icons/src/loading-icons.md?plain=1#L111

https://github.com/palantir/blueprint/blob/c8570835d81382313c2bcf3a09aafe614f3cd854/packages/icons/src/loading-icons.md?plain=1#L139

https://blueprintjs.com/docs/#icons/loading-icons.using-dynamic-imports